src/app/contextBroker/contextBroker.cpp

```diff
@@ -226,6 +227,7 @@ char            rush[256];
 #define DBUSER_DESC         "database user"
 #define DBPASSWORD_DESC     "database password"
 #define DB_DESC             "database name"
+#define TIMEOUT_DESC        "replica set timeout"
```

Comment:
A bit more detail here would be great.

"timeout in seconds for connection to database" ?

On the other hand, 'replica set' is perhaps too detailed, and might not inform people what it is really about  :-)